golang/pkg/cli/container_defaults.go

```diff
@@ -596,3 +613,44 @@ func CopyTraefikConfiguration(ctx context.Context, name, internalHostDomain stri
 
 	return err
 }
+
+func healhProbe(ctx context.Context, address string) error {
+	req, err := http.NewRequestWithContext(ctx, http.MethodGet, address, http.NoBody)
+	if err != nil {
+		return fmt.Errorf("failed to create the http request: %s", err.Error())
+	}
+
+	log.Info().Str("address", address).Msg("Health probing")
+
+	startedAt := time.Now()
+	timeOut := false
+	statusCode := -1
+	for !timeOut {
+		//nolint:bodyclose //closed already
+		resp, reqErr := http.DefaultClient.Do(req)
+
+		if time.Since(startedAt) >= healhProbeTimeout {
```

Comment:
Indeed, and it has been like that before. I had to change it, because it was failing with a DNS resolution error, which of course hasn't been taken into account by the timeout mechanism.

I change it back to the original one.